docs/_includes/top.html

```diff
@@ -1,20 +1,34 @@
 {% assign top = page.url | slice: 1, 3 %}
-
-<div id="top">
-
-<div id="top-left" class="flex1">
-<a href="/" class="windowjs top-selected">Window.js</a>
-</div>
-
-<div id="top-center">
-<a href="/about" {% if top == "abo" %}class="top-selected"{% endif %}>About</a>
```

Comment:
If "top-selected" isn't used anymore then it can be removed from style.css

Reply:
Yes, the top navigation bar will rely on Bootstrap classes to set the style. "top-selected" will be removed from styles.css